packages/ui/src/components/option.tsx

```diff
@@ -32,14 +32,14 @@ export const Option: FC<OptionProps> = ({ control, id, label, description, ariaS
       aria-labelledby={`${id}-label`}
       aria-selected={ariaSelected}
     >
-      {control}
+      <div className="mt-0.5">{control}</div>
       <div className="flex flex-col gap-0">
-        <Label htmlFor={id} className="cursor-pointer pl-2.5 leading-tight">
+        <Label htmlFor={id} className="mb-1 cursor-pointer pl-2.5" variant="primary">
```

Comment:
If we are not adding default margin to `Label`, can we remove `mb-1` and add `gap-1` to its parent? Also can we add `pl-2.5` to its parent and remove it from here and `ml-2.5` from the `Text` below?